mods/ra/rules/palettes.yaml

```diff
@@ -40,7 +40,12 @@
 	PaletteFromFile@effect:
 		Name: effect
 		Filename: temperat.pal
-		ShadowIndex: 4
+		ShadowIndex: 4, 15
```

Comment:
This appears to have broken some things. Now dragon missiles are black, flame thrower projectiles (fb3) also have black artifacts